pkgs/development/compilers/swift/default.nix

```diff
@@ -27,6 +32,15 @@ let
         llvmPackages_15.clang.override rec {
           libc = apple_sdk.Libsystem;
           bintools = pkgs.bintools.override { inherit libc; };
+          # Ensure that Swift’s internal clang is using the same libc++ and libc++abi
+          # used by the default Darwin stdenv.
+          libcxx = pkgs.llvmPackages.libcxx;
+          extraPackages = [
+            pkgs.llvmPackages.libcxxabi
+            (pkgs.llvmPackages_15.compiler-rt.override {
+              inherit (pkgs.llvmPackages) libcxxabi;
+            })
+          ];
         }
```

Comment:
Now that `llvmPackages_15` is repeated twice, can we turn this into a function, something like:
```
if pkgs.stdenv.isDarwin then
  # Comments here
  let makeSwiftClang = stdenvLLVM: targetLLVM:
      ...
  in
  makeSwiftClang pkgs.llvmPackages llvmPackages_15

Reply:
How’s the current push look like with the alias (`swiftLlvmPackages`)? I can still rework it to use a function, but it’s only going to be used right away, so it seems like it would add a limited amount of clarity.